students/drovdahl/lesson08/circle.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/local/bin/python3
+'''
+Class that does the following:
+    Compute the circle's area
+    Print the circle and get something nice
+    Add two circles together
+    Compare two circles to see which is bigger
+    Compare two circles to see if they are equal
+    Put circles in a list and sort them
+
+Use the following techniques
+    properties
+    classmethod
+    special methods
+'''
+import math
+
+
+class Circle(object):
+    def __init__(self, radius):
+        self._radius = radius
+        self.diameter = radius * 2
+
+    @property
+    def radius(self):
+        return self.diameter / 2
```

Comment:
Why not just `return self._radius`?

Reply:
Not sure why I did that...  Probably because it was 1AM . :)

Reply:
This is addressed in my next PR